bin/upgradeversion

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Check postgres version
+echo -n "Current database version is "
+cat ${PGDATAOLD}/PG_VERSION
+
+# Abort if the old version is not 16
+if [ "$(cat ${PGDATAOLD}/PG_VERSION)" != "16" ]; then
+    echo "Only PostgreSQL 16 is supported for migration with this image."
+    exit 1
+fi
+
+chown -R postgres:postgres "${PGDATAOLD}" "${PGDATANEW}"
+
+# Initialize the new database, if it doesn't exist
+if [ ! -s "${PGDATANEW}/PG_VERSION" ]; then
+    gosu postgres initdb -D ${PGDATANEW}
```

Comment:
We can't use gosu because all containers start with a static UID and any cap dropped. In this case 999. You can test the container with docker run -u 999 ... . Maybe you have to delete the volume fist.
https://github.com/greenbone/automatix/blob/main/compose/dev/components/opensight/asset-management/compose.yaml#L14